packages/webapp/src/App.tsx

```diff
@@ -140,17 +140,17 @@ const App = () => {
                         <Route path="/:env/project-settings" element={<ProjectSettings />} />
                     </Route>
                     <Route path="/auth-link" element={<AuthLink />} />
-                    {(isCloud() || isEnterprise()) && (
+                    {(isCloud() || isEnterprise() || isLocal()) && (
```

Comment:
when is this condition not true anymore now?

Reply:
hosted